test-apps/display-test-app/.gitignore

```diff
@@ -1,2 +1,6 @@
 Package.resolved
 .env.local.mobile
+.vite-inspect/
+public/
```

Comment:
Why are you ignoring everything in the public dir?

Reply:
Vite needs all static assets from dependencies to be in the public folder of the current directory, so they are copied during the build process with the copy plugin. The public directory is ignored to avoid committing the copied files.

Reply:
The public directory contains many files that need to remain under source control. If your PR is now also treating "public" as an output directory then it should either use a different directory (does vite require that the public directory be named "public"?) or the existing public directory should be renamed and its contents copied to your output directory during build.

Reply:
Good call, I copied the files into a new hidden directory that will be ignored.